readme.md

```diff
@@ -253,6 +253,20 @@
 | ------: | -------: | -----: |
 |   1 - 5 |       10 |40 -70m |
 
+### [The Quacks of Quedlinburg](https://boardgamegeek.com/boardgame/244521/quacks-quedlinburg)
+
+> In The Quacks of Quedlinburg, players are charlatans — or quack doctors — each making their own secret brew by adding ingredients one at a time. Take care with what you add, though, for a pinch too much of this or that will spoil the whole mixture!
+> 
+> Each player has their own bag of ingredient chips. During each round, they simultaneously draw chips from their bags and add them to their pots. The higher the face value of the drawn chip, the further it is placed in the pot's swirling pattern, increasing how much the potion will be worth. Push your luck as far as you can, but if you add too many cherry bombs, your pot will explode!
+> 
+> At the end of each round, players gain victory points and coins to spend on new ingredients, depending on how well they managed to fill up their pots. But players whose pots have exploded must choose points or coins — not both! The player with the most victory points at the end of nine rounds wins the game.
+
+![The Quacks of Quedlinburg game image](https://cf.geekdo-images.com/pH5LgRL4mNRon-2NsSDb1Q__itemrep/img/qe9ywOTxtPr_n4DoXZgEY-C2qU0=/fit-in/246x300/filters:strip_icc()/pic6137509.png)
+
+| Players | Min. Age | Time   |
+| ------: | -------: | -----: |
+|   2 - 5 |        8 | 30-60m |
```

Comment:
```suggestion
|   2 - 4 |       10 |  45m |
```

Please correct the stats blcok.

Reply:
@edm00se, I've implemented the required modifications. Would you please review the PR and proceed with the merge? Thank you!

Reply:
> Please correct the stats block. We seek to be in agreement with Board Game Geek's listing (which is usually accurate to the publisher's).

@edm00se, I've implemented the required modifications. Would you please review the PR and proceed with the merge? Thank you!